.travis.yml

```diff
@@ -8,6 +8,11 @@ env:
 
 before_install:
   - travis_retry go mod vendor
+  - go get golang.org/x/lint/golint
+
+script:
+  - golint $(go list ./... | grep -v /vendor/)
```

Comment:
Missing the `-set_exit_status` command line flag.

This command should be `golint -set_exit_status $(go list ./... | grep -v /vendor/)`